tools/accuracy_checker/accuracy_checker/adapters/detection.py

```diff
@@ -897,9 +897,9 @@ def box_cxcywh_to_xyxy(x):
                  (x_c + 0.5 * w), (y_c + 0.5 * h)]
             return b
 
-        def softmax(x):
-            exp_x = np.exp(x)
-            return exp_x / np.sum(exp_x)
+        def softmax(logits):
```

Comment:
why do you need this?

Reply:
Softmax is applied along entire scores vector in the previous implementation, but in the original version softmax is applied to values along last dim https://colab.research.google.com/github/facebookresearch/detr/blob/colab/notebooks/detr_demo.ipynb#scrollTo=BiwSmd2i-Wkf